epublib-core/src/test/java/nl/siegmann/epublib/epub/ResourcesLoaderTest.java

```diff
@@ -143,36 +134,36 @@ public void testLoadResources_ZipFile_partial_lazy() throws FileNotFoundExceptio
 	private void verifyResources(Resources resources) throws IOException {
 		Assert.assertNotNull(resources);
 		Assert.assertEquals(12, resources.getAll().size());
-		List<String> allHrefs = new ArrayList<String>(resources.getAllHrefs());
-		Collections.sort(allHrefs);
+		List<String> allHRefs = new ArrayList<>(resources.getAllHrefs());
```

Comment:
typo: Please rename to allHrefs

Reply:
It was a change suggested by IntelliJ. It seems to like allHRefs better :)
Reverted.